javascript/src/URDFLoader.js

```diff
@@ -322,10 +322,12 @@ class URDFLoader {
 
                 } else if (type === 'texture') {
 
-                    const loader = new THREE.TextureLoader(manager);
                     const filename = n.getAttribute('filename');
-                    const filePath = resolvePath(filename);
-                    material.map = loader.load(filePath);
+                    if(filename && filename != "") {
```

Comment:
This line is a bit redundant because if filename is `null`, `""`, or `undefined` it will be falsey so this can just be changed to

```js
if (filename) {
// ...
}
```